### DISCUSSION TITLE
Own repository for product-pictures

### DISCUSSION THREAD

**Discussion by Burki24** [Koenkk/zigbee2mqtt#13677] [General]
@Koenkk 

Will it be possible, to add a configuration-field for own device-picture directory? 

I add newly the TV05-ZG curve TRV, It's based on the TV02-Zigbee-TRV. So I only added the modelID and manufacturerName. But it lokks complete different to the TV02:

TV02: ![image](https://user-images.githubusercontent.com/20765982/186573697-a5830e7c-ff70-4801-993c-cca92390a208.png)
TV05: ![image](https://user-images.githubusercontent.com/20765982/186573807-48a393ad-1c98-4db2-b7df-93586d1d5fe8.png)

So some device-pictures want to be individuell changed. That will be a great enhancement for us.

Thanks in advance
Burkhard

**Comment by Koenkk** [comment]
That is not possible, if it is a different device I suggest to create a separate definition. Then it will also appear correctly on the supported device page.

**Comment by Burki24** [comment]
okay, thanks. It's not a different device, only a different design.

**Comment by Burki24** [comment]
@Koenkk 

I will come back to this topic as I tried the "Icon" field in Device->Settings->Icon.
When I place an url like "https://github.com/Burki24/Z2M_Pictures/blob/main/devices/radar-wall-1.jpg", it will not be loaded, also not on restart Zigbee2MQTT.

in devices.yaml it looks correct:
```code
'0xe0798dfffed6de48':
  friendly_name: Wohnzimmer/Sicherheit/Wohnbereich/Radar
  homeassistant: {}
  icon: https://github.com/Burki24/Z2M_Pictures/blob/main/devices/radar-wall-1.jpg
  legacy: false
  optimistic: false
```


The picture is available and also in .jpg 150x150. Are there more things, we have to do?

Many greetings from Berlin
Burkhard

**Reply by Koenkk** [reply]
Nowadays we can correct the picture based on the manufacturerName, please fill in https://github.com/Koenkk/zigbee2mqtt/issues/new?assignees=&labels=wrong+device&template=wrong_device.yaml&title=%5BWrong+device%5D%3A+

**Reply by Burki24** [reply]
Thanks for this hint. I made it as you said.
